build/localnet/README.md

```diff
@@ -0,0 +1,165 @@
+# Kubernetes LocalNet <!-- omit in toc -->
+
+This guide shows how to deploy a LocalNet using [pocket-operator](https://github.com/pokt-network/pocket-operator).
```

Comment:
@okdas I updated most of this file while testing so please review.

Reply:
Looks good, I also added some changes (mostly requested ones). Please lmk if you have questions.